src/Moq/Protected/ProtectedAsMock.cs

```diff
@@ -239,11 +246,21 @@ protected override Expression VisitMethodCall(MethodCallExpression node)
 				{
 					var targetParameter = Expression.Parameter(this.targetType, left.Name);
 					return Expression.Call(targetParameter, FindCorrespondingMethod(node.Method), node.Arguments);
-				}
+				} 
```

Comment:
Detail: please revert this whitespace-only change.

Reply:
done